### ISSUE TITLE
Closed window with open folder/project keeps getting brought back over untitled empty window

### ISSUE THREAD

**Issue by wzxu** [zed-industries/community#1047]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

1. Open a folder in current window
2. Close current window
3. Open a new empty window
4. quit & re-launch Zed

Expectation: Zed will restore the lastly opened window, aka the empty window
Actual: Zed brings back the closed window with open folder/project

### Environment

Zed: v0.75.2 (stable)
OS: macOS 13.2.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by b-fuze** [comment]
Yeah, I was also trying to find out how to remove the project and restore the empty window on startup and couldn't figure out how
